scripts/vcf_filter.py

```diff
@@ -108,7 +108,7 @@ def addfilt(filt):
     while len(args.rest):
         filter_name = args.rest.pop(0)
         if filter_name not in filters:
-            sys.exit("%s is not a known filter (%s)" % (filter_name, str(filters.keys())))
+            sys.exit("%s is not a known filter (%s)" % (filter_name, str(list(filters.keys()))))
```

Comment:
Maybe use
```
from dcicutils.lang_utils import disjoined_list
```
and then
```suggestion
            sys.exit(f"{filter_name} is not a known filter. Must be one of {disjoined_list(filters)}.")
```
It will use the keys automatically. Even if you don't use `disjoined_list`, the call to `str` seems unnecessary, and the rewrite to an f-string would be easier to read. But, for reference:
```
>>> from dcicutils.lang_utils import disjoined_list
>>> disjoined_list({'a': 3, 'b': 4})
'a or b'
```

Reply:
OK, can't use dcicutils.